README.md

```diff
@@ -86,5 +95,7 @@ The most common mood is :-)
 ```
 
 **think**: Should you use an array or a hash to solve this problem? Why?
+Because we were simply tallying the data and not trying to find specific days, I used an array to count the happy and sad days. I stored the dates of the happy and sad results instead of just tallying how many there were because it seemed like a useful piece of data to be able to access (which days were sad, which were happy, would be easy to access).
```

Comment:
This reasoning is well supported, but I would have like to see that data stored somewhere where it could be used later, for this justification.